.trampolinerc

```diff
@@ -0,0 +1,63 @@
+# Copyright 2020 Google LLC
```

Comment:
```suggestion
# Copyright 2023 Google LLC
```

Reply:
This will be fixed at the source in https://github.com/googleapis/synthtool/pull/1815